app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -146,6 +157,10 @@
     // latest configuration
     AaaConfig newCfg;
 
+    ScheduledExecutorService ses = Executors.newScheduledThreadPool(1);
+    ScheduledFuture<?> scheduledFuture;
+
+    List<Byte> outPacketList = new ArrayList<Byte>();
```

Comment:
Use a HashSet instead of List as each entry will be unique and you need to lookup, where Set is more suitable